tests/scenarios/signing/ExportCertificate.ps1

```diff
@@ -0,0 +1,24 @@
+param (
+	[Parameter(Mandatory=$True)]
+	[string]$certFriendlyName,
+	
+	[Parameter(Mandatory=$True)]
+	[string]$CertStoreLocation
+)
+
+function TryGetCert
+{
+	write-host ("Cert friendly name is: " + $certFriendlyName)
+    $results = Get-ChildItem "$CertStoreLocation" | Where-Object { $_.FriendlyName -eq "$certFriendlyName" }
+    if ($results.Count -gt 0)

```

Comment:
Maybe spit out a warning if the count is more than 1? Would be an amibuous cert resolution.